ui/page/components/account_selector.go

```diff
@@ -200,39 +205,59 @@ func (as *AccountSelector) Layout(gtx layout.Context) layout.Dimensions {
 }
 
 func (as *AccountSelector) ListenForTxNotifications(ctx context.Context) {
+	// Complain if it try to listen for TX Notifications
+	// When not subscribed.
+	if !as.subbedTxNotif {
+		log.Warn("Account Selector not subscribed for Tx notifications but listening")
+	}
```

Comment:
Oh i get it `subbedTxNotif` is subscribed? 
Thinking again, i think `IsTxNotificationEnabled` is better.

I don't think this is needed as the users do not get to see the message anyways to act on it. Plus it is not an error.

Reply:
My idea is not to have it there for the user. It is there for debugging, just like the rest of the debugging messages.

Reply:
Question is what are you debugging? in `SubscribeTxNotifications()`, you log sub and the set `subbedTxNotif` to true. 

I believe the `subbedTxNotif` is actually redundant and can be removed.